src/PrestaShopBundle/Bridge/Smarty/BreadcrumbsAndTitleConfigurator.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Bridge\Smarty;
+
+use Link;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShopBundle\Bridge\AdminController\ControllerConfiguration;
+use Tab;
+use Tools;
+
+/**
+ * This class get the breadcrumb and title information configuration from Tabs,
+ * and sets them in the controller configuration.
+ */
+class BreadcrumbsAndTitleConfigurator implements ConfiguratorInterface
+{
+    /**
+     * @var Link
+     */
+    private $link;
+
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * @param LegacyContext $legacyContext
+     * @param Configuration $configuration
+     */
+    public function __construct(LegacyContext $legacyContext, Configuration $configuration)
+    {
+        $this->link = $legacyContext->getContext()->link;
+        $this->configuration = $configuration;
+    }
+
+    /**
+     * Set breadcrumbs array for the controller page.
+     *
+     * @param ControllerConfiguration $controllerConfiguration
+     */
+    public function configure(ControllerConfiguration $controllerConfiguration): void
+    {
+        $tabs = [];
```

Comment:
Not needed as `Tab::recursiveTab` will return you an array already, right?

Reply:
You right